### PR TITLE
Add workflow to automate integration tests with staging

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject 

Part of https://github.com/ballerina-platform/ballerina-standard-library/issues/2933

### CODE REVIEW

File: .github/workflows/test_stdlib_releases_with_staging.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: Test Standard Library Releases with Staging Central
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test-patch-versions:
+    name: Test Pacth Standard Library Releases
```
Comment (ThisaruGuruge) [inline .github/workflows/test_stdlib_releases_with_staging.yml:8]:
```suggestion
    name: Test Patch Standard Library Releases
```

File: .github/workflows/test_stdlib_releases_with_staging.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      - name: Run Standard Library Tests w/ Staging
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+          devCentralToken: ${{ secrets.BALLERINA_DEV_CENTRAL_ACCESS_TOKEN }}
+        run: ./gradlew testStdlibsWithStaging --stacktrace --scan --console=plain --no-daemon --continue
+
+  test-minor-versions:
+    name: Test Pacth Standard Library Releases
```
Comment (ThisaruGuruge) [inline .github/workflows/test_stdlib_releases_with_staging.yml:37]:
```suggestion
    name: Test Patch Standard Library Releases
```

File: .github/workflows/test_stdlib_releases_with_staging.yml (hunk 1 above)
Comment (niveathika) [reply .github/workflows/test_stdlib_releases_with_staging.yml]: Fixed in [c5dcf5c](https://github.com/ballerina-platform/ballerina-standard-library/pull/3134/commits/c5dcf5cafb944c36435e02f97db358ffc56e865c)

File: .github/workflows/test_stdlib_releases_with_staging.yml (hunk 2 above)
Comment (niveathika) [reply .github/workflows/test_stdlib_releases_with_staging.yml]: Fixed in [c5dcf5c](https://github.com/ballerina-platform/ballerina-standard-library/pull/3134/commits/c5dcf5cafb944c36435e02f97db358ffc56e865c)

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**